js/colourpicker.js

```diff
@@ -153,16 +143,22 @@
       if( input.data('colourpicker-initialized') ) return;
 
       // Handle settings
-      settings = $.extend(true, {}, defaults, settings);
-
-	    // Palette type
-	    colourpicker.addClass('palette-' + settings.palette);
-	  
-	    // If adding a transparent checkbox, make the parent an input group
-      if( settings.allowTransparent ) {
-        colourpicker.addClass('input-group');
+      settings = $.extend(false, {}, defaults, settings);
+      //for backwards compatibility
+      if ($.type(settings.allowedCols) === "string") {
+        settings.allowedCols = settings.allowedCols.split(" ");
       }
-
+	    // Palette type, alpha or not, make it an input group.
```

Comment:
This was the easiest way I could come up with to get a background image of the checkered box. I could add another container class instead, but since this one was already there I decided to leverage it and give input-group the checkered background. However, I see it caused some UI issues further down so I'll reconsider whether this makes sense.